<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>176</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ControlLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="Titlelabel">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Scope</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="UartLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="Comlabel">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>UART</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComBox">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="ComSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="BoudrateLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="BaudLabel">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Baudrate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="BaudBox">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="BaudSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="FlashLayout">
       <item>
        <widget class="QLabel" name="Flashlabel">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>FlashRate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FlashEdit">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenGL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <property name="text">
        <string>OpenGL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddChart">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
        </font>
       </property>
       <property name="text">
        <string>AddChart</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>961</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>426</y>
      <width>81</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="Xaccel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Xaccel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Yaccel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Yaccel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Zaccel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Zaccel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Xgyro">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Xgyro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Ygyro">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Ygyro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Zgyro">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Zgyro</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>427</y>
      <width>81</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="XaccelT">
       <property name="text">
        <string>XaccelT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="YaccelT">
       <property name="text">
        <string>YaccelT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ZaccelT">
       <property name="text">
        <string>ZaccelT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="XgyroT">
       <property name="text">
        <string>XgyroT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="YgyroT">
       <property name="text">
        <string>YgyroT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ZgyroT">
       <property name="text">
        <string>ZgyroT</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
